tests/test.sh

```diff
@@ -1121,9 +1121,9 @@ function test-pytests()
 # #############################################################################
 function test-build-and-run-spdlog-sample()
 {
-    make clean && CC=g++ LD=g++ BUILD_VERBOSE=1 make spdlog-cpp-program
+    make -C tests clean && CC=g++ LD=g++ BUILD_VERBOSE=1 make -C tests spdlog-cpp-program
 
-    local test_prog="./build/${Build_mode}/bin/use-cases/spdlog-Cpp-program"
+    local test_prog="tests/build/${Build_mode}/bin/use-cases/spdlog-Cpp-program"
```

Comment:
Multiple such places where it would be nice[r] to replace: `./tests/build/.../`

Çan you please fix all such instances?